README.md

```diff
@@ -231,9 +231,9 @@ and with a smooth evolutionary path.
 ## Memory safety
 
 Safety, and especially
-[memory safety](https://en.wikipedia.org/wiki/Memory_safety), remain key
-challenges for C++ and something a successor language needs to address. Our
-initial priority and focus is on immediately addressing important, low-hanging
+[memory safety](https://en.wikipedia.org/wiki/Memory_safety), remains a key
+challenge for C++ and something a successor language needs to address. Our
+initial priority and focus are on immediately addressing important, low-hanging
```

Comment:
I find the "is" to "are" change here surprising, but my grasp of this part of the English grammar is quite weak so I'll trust others on this.

Reply:
so should I change it back to 'is'?

Reply:
No, I'll trust others' review here.

Reply:
that pre-commit got failed at 'prettier', what does it mean? I should format it with prettier on locally?